src/lib/cookies.ts

```diff
@@ -112,7 +114,7 @@ function serialize(
  */
 function isSecureEnvironment(req: any) {
   if (!req || !req.headers || !req.headers.host) {
-    throw new Error('The "host" request header is not available')
+    throw new AuthUnknownError('The "host" request header is not available', new Error('The "host" request header is not available'))
```

Comment:
You have an `new Error` inside of your `new AuthUnknownError`.

Reply:
yeah just passing it through because the `AuthUnknownError` takes in a message and an error, since this code path doesn't have an actual underlying error, i'm just creating one and wrapping it with the `AuthUnknownError` class

Reply:
Think this is intended -- `AuthUnknownError` [seems to take in a string and constructor](https://github.com/supabase/gotrue-js/pull/341/files#diff-33a1fa952d3520ae3b8ff121fe89f8d401e10efd4cbf34f180663caa16c0f153L35)